### PR TITLE
fetch answers (backend)

### PR DESCRIPTION
**Pull request by mitalirs**
## Related Issuse

- There was no route that enabled a user to fetch all the answers to a particular question

Closes: #251 

## Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.(Specially for Backend)
- [x] My changes generate no new warnings.

## Screenshots

|        Original         |          Updated           |
| :---------------------: | :------------------------: |
| **no getAnswers route** | <b> Screenshot (411) </b> |

### CODE REVIEW

File: server/API DOCS/GET.md [hunk 1]
```diff
@@ -17,5 +17,24 @@
     * **Code:** `401` <br />
       **Message:** `authorization required` <br />
 
+* **Required Filed**
+    * **Auth/ Admin Auth: true/true**
```
Comment (codewithvk) [inline server/API DOCS/GET.md:21]: Here not need to be admin auth so you can write like this.

`Auth/ Admin Auth: true/false `  (First for user Auth/second for admin auth)

File: server/API DOCS/GET.md [hunk 2]
```diff
@@ -17,5 +17,24 @@
...
+    `GET`
+
+* **Success Response**<br />
+    * **Code:** `200` <br />
+      **data:** `{ answers: [ displaying all answers to a particular question] }`
+
+* **Error Response**
+    * **Code:** `400` <br />
+      **Message:** `something went wrong` <br />
+
```
Comment (codewithvk) [inline server/API DOCS/GET.md:38]: Here not need to be admin auth so you can write like this.

Auth/ Admin Auth: true/false (First for user Auth/second for admin auth)

**Review by codewithvk — CHANGES_REQUESTED** [review]

**Comment by codewithvk** [comment]
@mitalirs I have Update docs so you get more idea about it.

File: server/API DOCS/GET.md (hunk 2 above)
Comment (mitalirs) [reply server/API DOCS/GET.md:39]: I have made a change, is that fine?

Comment (codewithvk) [reply server/API DOCS/GET.md:39]: Yup!

**Review by codewithvk — APPROVED** [review]

**Pull request merged**